notifications_utils/recipients.py

```diff
@@ -165,9 +165,21 @@ def _rows(self):
             skipinitialspace=True,
         )
 
+    @property
+    def _first_empty_column_indices(self):
+        for row_index, row in enumerate(self._rows):
+            if row_index == 0:
+                continue  # skip the header row
+            yield max(column_index for column_index, column in enumerate(row) if column) + 1
```

Comment:
Does this choke on an empty row? (though perhaps in reality it chokes elsewhere first?)